packages/metal-state/test/State.js

```diff
@@ -1041,6 +1041,32 @@ describe('State', function() {
 			assert.strictEqual('foo2', test2.key1);
 			assert.strictEqual('foo1', test1.key1);
 		});
+
+		it('should configure static STATE with multiple levels of class inheritance', function() {
+			var Test = createTestClass();
+			Test.STATE = {
+				key1: {
+				}
+			};
+
+			class Child extends Test {
+			}
+			Child.STATE = {
+				key2: {
+				}
+			};
+
+			var test = new Test({
+				key1: 'foo1'
+			});
+			assert.strictEqual('foo1', test.key1);
+
+			var child = new Child({
+				key2: 'foo2'
+			});
+			assert.strictEqual('foo1', test.key1);
```

Comment:
Shouldn't this be asserting `child.key1`?

Reply:
Actually, if you change it to `child.key1`, the test will fail...

Reply:
No, it's shouldn't be asserting `child.key1`, key1 isn't being passed a value for `Child`. I just left this assertion there to ensure that the static hint configuration of `Child` doesn't affect the values of the parent it's inheriting from.

Reply:
Ah, I see! For some reason I assumed they had defaults!